### ISSUE TITLE
[Proposal]: move target mesh drawing options from Display to Target Visibility

### ISSUE THREAD

**Issue by vxlcoder** [CGCookie/retopoflow#1008]
currently, the target mesh drawing options (alpha, normal offsets, etc) are under display, but we also have target visibility options (hide/reveal)

![image](https://user-images.githubusercontent.com/3790699/121896317-2a11ed00-ccef-11eb-9112-81e6e272229e.png)

this could cause some confusion, especially when the artist is not able to see the target geometry (ex: target is below surface / source mesh pushes through target mesh).

my suggestion is to move the target mesh drawing options to be under target visibility, and add a note to top of Target Visibility section that these options only change the way the target is visualized, and see Target Cleaning to make changes to target geometry.

note: this proposal will keep the auto hide tool options, hide overlays, theme, view clipping, etc. all under display still.  a related proposal: move display section to be under general similar to tooltips and advanced (how often do these options really need touched??)

**Comment by jlampel** [comment]
I did some organizing here, but I didn't find a way to do this proposal cleanly. I think there's enough of a separation between hide/unhide actions and the setting for drawing the mesh itself. If we moved the mesh drawing options down then would we move the non-manifold outline down with it or up to general? Would the clip settings be in general or target visibility? That one doesn't seem to make sense in either. 

My proposal is to move all the RF UI specific stuff to General and then keep Display for all the 3D drawing settings. 
![image](https://user-images.githubusercontent.com/12982300/122251659-8efc4d00-ce7f-11eb-8fee-219be74505ba.png)

I'm definitely open changing it up, but I'd like to keep similar things grouped together without any one section getting too huge, so that users can work with multiple sections open at once.
